common/src/main/scala/com/gu/media/aws/AwsCredentials.scala

```diff
@@ -0,0 +1,58 @@
+package com.gu.media.aws
+
+import com.amazonaws.auth._
+import com.amazonaws.auth.profile.ProfileCredentialsProvider
+import com.amazonaws.services.securitytoken.AWSSecurityTokenServiceClientBuilder
+import com.gu.media.Settings
+
+case class AwsCredentials(instance: AWSCredentialsProvider, crossAccount: AWSCredentialsProvider,
+                          upload: AWSCredentialsProvider)
```

Comment:
I disagree. We also use kinesis streams within media-service for direct upload, so I want to make a clear distinction between what is local and cross-account